lib/content_exporter.rb

```diff
@@ -0,0 +1,44 @@
+require "csv"
+
+class ContentExporter
```

Comment:
This is a class with only static methods, it's probably better to use a module with module functions.

Reply:
What are the advantages to module functions over a class with static methods?
I'm not particularly attached to this way of doing things, just seemed like a convenient hack.

What do module functions get me instead?

Reply:
:thinking: thinking about this humm... i guess a class has very little meaning if you can't make an instance right?
So perhaps that's why i should use `module` here instead...

Reply:
Humm had a read through this: https://dvelp.co.uk/articles/ruby-class-vs-module
I think it's making sense now:
> As a rule of thumb, if you create a Class and it has no instance methods,
you shouldn't be using a Class

Will switch to module now

Reply:
Done here: https://github.com/alphagov/govuk-coronavirus-find-support/pull/216/commits/305ed15f2f834d9f2a738c3923d42c7916eada0b